demisto_sdk/commands/common/constants.py

```diff
@@ -221,21 +219,344 @@ def no_common_server_python(path):
+        return f'DA100 - {file_path}: the field {field} needs to be removed.\n'
+
+    @staticmethod
+    def remove_field_from_widget(field, widget):
+        return f'WD100 - The field {field} needs to be removed from the widget: {widget}.\n'
+
+    @staticmethod
+    def include_field_in_dashboard(file_path, field):
+        return f'DA101 - {file_path}: the field {field} needs to be included. Please add it.\n'
+
+    @staticmethod
+    def include_field_in_widget(field, widget_name):
+        return f'WD101 - The field {field} needs to be included in the widget: {widget_name}. Please add it.\n'
+
+    @staticmethod
+    def description_missing_in_beta_integration(package_path):
+        return f"DS100 - No detailed description file was found in the package {package_path}. Please add one, " \
+               f"and make sure it includes the beta disclaimer note." \
+               f"It should contain the string in constant\"BETA_INTEGRATION_DISCLAIMER\""
+
+    @staticmethod
+    def no_beta_disclaimer_in_description(package_path):
+        return f"DS101 - Detailed description in beta integration package {package_path} " \
+               f"dose not contain the beta disclaimer note. It should contain the string in constant" \
+               f"\"BETA_INTEGRATION_DISCLAIMER\"."
+
+    @staticmethod
+    def no_beta_disclaimer_in_yml(file_path):
+        return f"DS102 - Detailed description field in beta integration {file_path} " \
+               f"dose not contain the beta disclaimer note. It should contain the string in constant" \
+               f" \"BETA_INTEGRATION_DISCLAIMER\"."
+
+    @staticmethod
+    def description_in_package_and_yml(package_path):
+        return f"DS103 - A description was found both in the " \
+               f"package and in the yml, please update the package {package_path}."
+
+    @staticmethod
+    def default_docker_error():
+        return 'DO100 - The current docker image in the yml file is the default one: demisto/python:1.3-alpine,\n' \
+               'Please create or use another docker image\n'
+
+    @staticmethod
+    def latest_docker_error(docker_image_tag, docker_image_name):
+        return f'DO101 - "latest" tag is not allowed,\n' \
+               f'Please create or update to an updated versioned image\n' \
+               f'You can check for the most updated version of {docker_image_tag} ' \
+               f'here: https://hub.docker.com/r/{docker_image_name}/tags\n'
+
+    @staticmethod
+    def not_demisto_docker():
+        return 'DO102 - docker image must be a demisto docker image. e.g: demisto/<image>:<tag>'
+
+    @staticmethod
+    def docker_tag_not_fetched(docker_image_name):
+        return f'DO103 - Failed getting tag for: {docker_image_name}. Please check it exists and of demisto format.'
+
+    @staticmethod
+    def no_docker_tag():
+        return 'DO104 - The docker image in your integration/script does not have a tag, please attach the latest tag'
```

Comment:
take the explanation from  - we don't want `latest`
f'DO101 - "latest" tag is not allowed,\n' \
               f'Please create or update to an updated versioned image\n' \
               f'You can check for the most updated version of {docker_image_tag} ' \
               f'here: https://hub.docker.com/r/{docker_image_name}/tags\n'

Reply:
done